web/src/app/core/types/host-component.ts

```diff
@@ -38,7 +38,7 @@ export interface HostComponent {
 /**
  * A service component that may be installed on hosts in a cluster.
  */
-export interface Component {
+export interface IComponent {
```

Comment:
What criterion do you choose for the prefix of an interface?

Reply:
I - interface
UpperCase for class